TruePath/AbsolutePath.cs

```diff
@@ -54,6 +54,14 @@ private AbsolutePath(string value, bool checkAbsoluteness)
     /// <inheritdoc cref="IPath.FileName"/>
     public string FileName => Underlying.FileName;
 
+    /// <summary>
+    /// Gets the current working directory as an AbsolutePath instance.
+    /// </summary>
+    /// <value>
+    /// The current working directory.
+    /// </value>
+    public AbsolutePath CurrentWorkingDirectory => new(Environment.CurrentDirectory);
```

Comment:
That's a nice addition!

Could you please make it `static`?

Reply:
solved in [c847c63](https://github.com/ForNeVeR/TruePath/pull/68/commits/c847c6320634e0b39407e1a9171b3f082cdf68a3)